gulpfile.babel.js

```diff
@@ -148,6 +148,27 @@ gulp.task('safari', ['safari:js'], () => {
   )
 })
 
+// Edge
+gulp.task('edge:template', () => {
+  return buildTemplate({SUPPORT_FILE_ICONS: true})
```

Comment:
You mentioned in the issue file icons aren't working. If that's the case, should mark this as 'false' until there's a fix. That being said, I'd love to have that fixed before merging, unless Edge can't really support fonts.

Reply:
The support configuration is removed for Edge. Actually, this is caused by an issue impacting how Edge handles Content Security Policy headers, which blocks the loading of icon fonts. This issue is fixed but not flighted yet. (Related Issue: [https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/11320212/]())